src/main/java/org/gridsuite/study/server/StudyService.java

```diff
@@ -1616,5 +1616,14 @@ public UUID getStudyUuidFromNodeUuid(UUID nodeUuid) {
     Mono<Void> updateBuildStatus(UUID nodeUuid, BuildStatus buildStatus) {
         return networkModificationTreeService.updateBuildStatus(nodeUuid, buildStatus);
     }
+
+    public Mono<Void> deleteModification(UUID nodeUuid, UUID modificationUuid) {
+        return networkModificationTreeService.getModificationGroupUuid(nodeUuid).flatMap(groupId ->
+                networkModificationService.deleteModification(groupId, modificationUuid)
+            )
+            .doOnSuccess(
```

Comment:
What is the `BuildStatus` if it fails for some `groupId` in map?

Reply:
we don't build network we invalidate

Reply:
You mean if there is an error when deleting modification, the network stays valid (if it was already valid) ?